### ISSUE TITLE
/usr/bin/whatsapp does not delete Service Worker

### ISSUE THREAD

**Issue by nerijus** [cycool29/whatsapp-for-linux#3]
It contains:
```
rm -rf ~/.config/whatsapp-nativefier-*/Service Worker
```
while it should be:
```
rm -rf ~/.config/whatsapp-nativefier-*/Service\ Worker
```
i.e. with `\` before space.
BTW, actually whatsapp works even without deleting Service Worker directory here.

**Comment by nerijus** [comment]
The following patch works:
```
diff --git a/build-whatsapp.sh b/build-whatsapp.sh
index ffb6af5..d397ae5 100644
--- a/build-whatsapp.sh
+++ b/build-whatsapp.sh
@@ -34,7 +34,7 @@ for FOLDER in $(ls | grep "^WhatsApp"); do
      cp -a ../${FOLDER} ./opt/WhatsApp
 
      echo '#!/bin/bash
-rm -rf ~/.config/whatsapp-nativefier-*/Service Worker && /opt/WhatsApp/WhatsApp' >./usr/bin/whatsapp
+rm -rf ~/.config/whatsapp-nativefier-*/Service\ Worker && /opt/WhatsApp/WhatsApp' >./usr/bin/whatsapp
 
     chmod +x ./usr/bin/whatsapp
 
```

**Comment by cycool29** [comment]
> It contains:
> 
> ```
> rm -rf ~/.config/whatsapp-nativefier-*/Service Worker
> ```
> 
> while it should be:
> 
> ```
> rm -rf ~/.config/whatsapp-nativefier-*/Service\ Worker
> ```
> 
> i.e. with `\` before space. 

Thanks, it is fixed in release 2.0.

> BTW, actually whatsapp works even without deleting Service Worker directory here.

It is to solve WhatsApp prompting it needs Chrome 60+ when using the app sometimes, see https://github.com/nativefier/nativefier/issues/1112.
